flood_risk_engine.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |s|
   # which details of the last email sent by the app can be accessed
   s.add_dependency "defra_ruby_email", "~> 1"
   s.add_dependency "defra_ruby_validators"
+  s.add_dependency "faraday-retry"
```

Comment:
Shouldn't this be a Gemfile dependency?

Reply:
Perhaps. Do you mean just for :development?

Reply:
in my understanding, faraday is only used by changelog gem, which isn't used in the runtime